sphinx_gallery/binder.py

```diff
@@ -105,18 +104,30 @@ def gen_binder_rst(fpath, binder_conf, gallery_conf):
     rst : str
         The reStructuredText for the Binder badge that links to this file.
     """
-    binder_conf = check_binder_conf(binder_conf)
     binder_url = gen_binder_url(fpath, binder_conf, gallery_conf)
-
-    binder_logo = os.path.join(glr_path_static(), 'binder_badge_logo.svg')
-    binder_logo_path = os.path.relpath(binder_logo, gallery_conf['src_dir'])
-
+    # In theory we should be able to use glr_path_static for this, but Sphinx
+    # only allows paths to be relative to the build root. On Linux, absolute
+    # paths can be used and they work, but this does not seem to be
+    # documented behavior:
+    #     https://github.com/sphinx-doc/sphinx/issues/7772
+    # And in any case, it does not work on Windows, so here we just for each
+    # gallery copy the SVG to `images` and link to it there. This will make
+    # a few copies, and there will be an extra in `_static` at the end of the
+    # build, but it at least works...
+    physical_path = os.path.join(
+        os.path.dirname(fpath), 'images', 'binder_badge_logo.svg')
+    os.makedirs(os.path.dirname(physical_path), exist_ok=True)
+    if not os.path.isfile(physical_path):
+        shutil.copyfile(
```

Comment:
I think for the test `test_gen_binder_rst` we shouldn't copy the file over

Reply:
Nevermind, `isfile` is better than `os.path.exists` that I used.

Reply:
I'll try some `tmpdir` gymnastics...